app/src/ui/changes/commit-message.tsx

```diff
@@ -738,6 +742,30 @@ export class CommitMessage extends React.Component<
     )
   }
 
+  private renderSummaryLengthHint(): JSX.Element | null {
+    return (
+      <TooltippedContent
+        delay={0}
+        tooltip={
+          <>
+            <span className="title">
+              Great commit summaries contain fewer than 50 characters
+            </span>
+            <br />
+            <span className="description">
+              Place extra information in the description field.
+            </span>
```

Comment:
Could we use block elements here and get rid of the `br` please
```suggestion
            <div className="title">
              Great commit summaries contain fewer than 50 characters
            </div>
            <div className="description">
              Place extra information in the description field.
            </div>
```